src/webextension/background/message-ports.js

```diff
@@ -186,6 +188,9 @@ export default function initializeMessagePorts() {
     case "proxy_telemetry_event":
       return telemetry.recordEvent(message.method, message.object,
                                    message.extra);
+    case "proxy_telemetry_scalar":
```

Comment:
I don't see where this is actually used.

I think you included this for the sake of completeness.  That's ok if  if we think it's likely other scalars will need recording from the front-end (e.g., full editor view, doorhanger, etc).

If we keep this, it will need a test in `test/unit/background/message-ports-test.js`; otherwise we can remove it for now.